aws-datastore/src/main/java/com/amplifyframework/datastore/syncengine/PendingMutation.java

```diff
@@ -365,34 +364,17 @@ public String toString() {
         /**
          * A model-creation mutation.
          */
-        CREATE(MutationType.CREATE),
+        CREATE,
 
         /**
          * Any change to an already-existing model, that does not *delete* the model.
          */
-        UPDATE(MutationType.UPDATE),
+        UPDATE,
 
         /**
          * The removal of a previously-created model.
          */
-        DELETE(MutationType.DELETE);
-
-        private final MutationType apiMutationType;
```

Comment:
All of this stuff was not used. And, it required more Javadocs. So, I just got rid of it, instead.

Reply:
Yay for simplifcation!